Gemfile.lock

```diff
@@ -347,33 +342,33 @@ GEM
       netrc (~> 0.8)
     rspec-core (3.8.2)
       rspec-support (~> 3.8.0)
-    rspec-expectations (3.8.4)
+    rspec-expectations (3.8.6)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.8.0)
-    rspec-mocks (3.8.1)
+    rspec-mocks (3.8.2)
       diff-lcs (>= 1.2.0, < 2.0)
       rspec-support (~> 3.8.0)
-    rspec-rails (3.8.2)
+    rspec-rails (3.8.3)
       actionpack (>= 3.0)
       activesupport (>= 3.0)
       railties (>= 3.0)
       rspec-core (~> 3.8.0)
       rspec-expectations (~> 3.8.0)
       rspec-mocks (~> 3.8.0)
       rspec-support (~> 3.8.0)
-    rspec-support (3.8.2)
-    rubocop (0.74.0)
+    rspec-support (3.8.3)
+    rubocop (0.78.0)
       jaro_winkler (~> 1.5.1)
       parallel (~> 1.10)
       parser (>= 2.6)
       rainbow (>= 2.2.2, < 4.0)
       ruby-progressbar (~> 1.7)
       unicode-display_width (>= 1.4.0, < 1.7)
-    rubocop-rails (2.3.2)
+    rubocop-rails (2.4.0)
       rack (>= 1.1)
       rubocop (>= 0.72.0)
     ruby-progressbar (1.10.1)
-    ruby-vips (2.0.15)
+    ruby-vips (2.0.16)
```

Comment:
### [ruby-vips](http://github.com/libvips/ruby-vips)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.0.15 → 2.0.16

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **15** commits. See the full changes on [the compare page](https://github.com/libvips/ruby-vips/compare/v2.0.15...v2.0.16).

These are the first 10 commits:
- (602bcf1) [add notes on otp use](https://github.com/libvips/ruby-vips/commit/602bcf13f55ef458fdf12d70e8ffc41524f70370)
- (0530df5) [improve libname generation](https://github.com/libvips/ruby-vips/commit/0530df59d37b12a467661aaf001c045dd10795dd)
- (be73566) [fix rubocop](https://github.com/libvips/ruby-vips/commit/be735660b112ce2b74ae3deb89df32782865d95d)
- (3c12206) [version bump and changelog entry](https://github.com/libvips/ruby-vips/commit/3c122068e0778107cb1a2c4d8102ab3dd9275aea)
- (e8135da) [Merge pull request #207 from libvips/fix-libname](https://github.com/libvips/ruby-vips/commit/e8135da716c9d91024ae244d51356c569ac2d5e1)
- (78fcacf) [add region and region#fetch](https://github.com/libvips/ruby-vips/commit/78fcacf907bb7db016b6666a986ae0274b02b70b)
- (11dacf8) [Merge branch 'master' into add-region-fetch](https://github.com/libvips/ruby-vips/commit/11dacf8f5140af710e7a2f4361752e15c2e30822)
- (dde5541) [add tests for region](https://github.com/libvips/ruby-vips/commit/dde5541acc3e1b8727961f0dc860dc915bc0b115)
- (634a560) [test libvips version for region spec](https://github.com/libvips/ruby-vips/commit/634a560aad85d08e8ee9e6cc504ef198a4e73188)
- (9bbee76) [fix a rubocop fail](https://github.com/libvips/ruby-vips/commit/9bbee76196b6a9f1766b0a324a0a3d9f7b0e2b59)


</details>